modules/oadp-features.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+
+[id="oadp-features_{context}"]
+= OADP features
+
+OpenShift API for Data Protection (OADP) supports the following features:
+
+Backup::
+You can back up all objects in your cluster or you can filter them by type, namespace, or label.
++
+Velero backs up Kubernetes objects and internal images in an archive file. Velero backs up PV data by using the cloud provider snapshot API or the Container Snapshot Interface (CSI) API to create snapshots. The archive files and snapshots are copied to object storage.
```

Comment:
`The archive files and snapshots are copied to object storage` This line is not true. Normal snapshots are not stored to the object storage. They are stored on the respective cloud provider. And CSI snapshots are also stored on their respective storage provider. It is only the restic snapshots that are backed up to the object storage.